AsyncDisplayKit/ASImageNode.mm

```diff
@@ -37,12 +37,12 @@ @interface _ASImageNodeDrawParameters : NSObject
 // TODO: eliminate explicit parameters with a set of keys copied from the node
 @implementation _ASImageNodeDrawParameters
 
-- (id)initWithImage:(UIImage *)image
-             bounds:(CGRect)bounds
-             opaque:(BOOL)opaque
-      contentsScale:(CGFloat)contentsScale
-    backgroundColor:(UIColor *)backgroundColor
-        contentMode:(UIViewContentMode)contentMode
+- (instancetype)initWithImage:(UIImage *)image
+                       bounds:(CGRect)bounds
```

Comment:
It indented using spaces. The change is due to the 1st line becoming longer. The subsequent lines are further indented in order to align `:`'s